src/main/resources/story/web_app/Trello End To End Scenarios.story

```diff
@@ -0,0 +1,90 @@
+Description: Test demoing VIVIDUS capabilities for Web Applications
+
+
+Lifecycle:
+Examples:
```

Comment:
[improvement suggestion] In a new project I would store locators in a separate e.g. data folder in .table file.